### PR TITLE
Iv 1205 cleanup for rl10.0.0

### PR DESCRIPTION
**Pull request by lopaka**

### CODE REVIEW

File: rll/setup_hostname.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# 'database.io my.example.com' all set the hostname to 'my.example.com'.
+# If SERVER_HOSTNAME is empty, will maintain current hostname.
+
+set -ex
+
+if [ -n "$SERVER_HOSTNAME" ]; then
+  prefix=
+  suffix=
+
+  re='^([-A-Za-z0-9_][-A-Za-z0-9_.]*[-A-Za-z0-9_])'
```
Comment (douglaswth) [inline rll/setup_hostname.sh]: Let's just put `${BASH_REMATCH[0]}` instead of adding parens to the regex.

File: rll/setup_hostname.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# The server's hostname is set to the longest valid prefix or suffix of
+# this SERVER_HOSTNAME variable eg 'my.example.com V2', 'NEW my.example.com', and
+# 'database.io my.example.com' all set the hostname to 'my.example.com'.
+# If SERVER_HOSTNAME is empty, will maintain current hostname.
+
+set -ex
+
+if [ -n "$SERVER_HOSTNAME" ]; then
```
Comment (douglaswth) [inline rll/setup_hostname.sh]: Let's use double brackets (`[[`/`]]`) for this instead of single.

**Comment by tve** [comment]
Looks good to me, feel free to merge from my point of view. i did not look into the details of how you set the hostname.

**Comment by tve** [comment]
Note that the first release will not be 10.0.0 but the next number after the last rc, something like 10.0.4 or 10.0.5

**Pull request merged**